### PR TITLE
Fixed #517 | Added doc for "How add translations", added new strings to translation and added them to Sinhala translation.

### LINKED ISSUE #517: Add how to translate file in readme.md
### Expected results
Added how to translate file in readme.md

### Would you like to work on this
yes I am working

### PR DESCRIPTION
**Pull request by STEdirisuriya**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.

### Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link**
  
  http://rawgit.com/STEdirisuriya/labyrinth/master/index.html

- 
  Fixes https://github.com/fossasia/labyrinth/issues/517

### The problem I want to solve or the facility I want to improve is/are
-  Added doc for "How add translations"
-  Added new strings to translation and added them to Sinhala translation.

### CODE REVIEW

**Comment by STEdirisuriya** [comment]
@niccokunzmann Review needed

**Comment by niccokunzmann** [comment]
Hi, in the future, we want to add the documentation to the documentation html files.

**Comment by niccokunzmann** [comment]
I just saw: travis is failing.

File: js/translate.js [hunk 1]
```diff
@@ -2,11 +2,13 @@
...
+
     'language': 'Select your language',
-    'how': 'HOW TO PLAY',
-    'credit': 'CREDITS',
-    'game': 'GAME',
+    'how': 'How to play',
+    'credit': 'Credits',
+    'game': 'Game',
     'contribute': 'Contributors',
+    'language': 'Select your language ',
```
Comment (duttaditya18) [inline js/translate.js]: The key `language` already exists.

File: js/translate.js [hunk 2]
```diff
@@ -28,16 +30,32 @@ var arrlang = {
...
     'flaticon': 'flaticon.com',
+
   },
   'si': {
     'language': 'ඔබගේ භාෂාව තෝරන්න',
     'how': 'ක්‍රීඩා කරන්නේ කෙසේ ද',
     'credit': 'සම්මානය',
     'game': 'ක්‍රීඩාව',
     'contribute': 'දායකයින්',
+    'language': 'ඔබගේ භාෂාව තෝරන්න',
```
Comment (duttaditya18) [inline js/translate.js]: The key `language` already exists.

**Comment by STEdirisuriya** [comment]
@duttaditya18 fixed

**Comment by STEdirisuriya** [comment]
@niccokunzmann Please see, If it get travis fail please help me to resolve it

**Comment by niccokunzmann** [comment]
It works!

**Pull request merged**